po/zh_CN.po

```diff
@@ -909,7 +909,7 @@ msgstr "全称："
 
 #: donate.phtml:39
 msgid "IBAN:"
-msgstr ""
+msgstr "IBAN："
```

Comment:
https://zh.wikipedia.org/wiki/%E5%9B%BD%E9%99%85%E9%93%B6%E8%A1%8C%E8%B4%A6%E6%88%B7%E5%8F%B7%E7%A0%81
I hope, it is the right writing.